diff --git a/src/extract/ExtractBuiltin.ml b/src/extract/ExtractBuiltin.ml
index 86f58b1a..f8597176 100644
--- a/src/extract/ExtractBuiltin.ml
+++ b/src/extract/ExtractBuiltin.ml
@@ -757,11 +757,11 @@ let mk_builtin_funs () : (pattern * Pure.builtin_fun_info) list =
         mk_fun
           "core::slice::index::{core::slice::index::SliceIndex<usize, [@T], \
            @T>}::index"
-          ~extract_name:(Some "core_slice_index_Slice_index") ();
+          ();
         mk_fun
           "core::slice::index::{core::slice::index::SliceIndex<usize, [@T], \
            @T>}::index_mut"
-          ~extract_name:(Some "core_slice_index_Slice_index_mut") ();
+          ();
         mk_fun "alloc::slice::{[@T]}::to_vec"
           ~extract_name:(Some "alloc.slice.Slice.to_vec") ();
         mk_fun "alloc::vec::{alloc::vec::Vec<@T>}::with_capacity"
